src/signing.rs

```diff
@@ -58,40 +58,36 @@ pub struct SigningKey {
     pub(crate) verifying_key: VerifyingKey,
 }
 
+/// # Example
```

Comment:
I moved this at the top so multiple things can be tested to add unit coverage via doctest